google/cloud/storage/internal/rest/stub.cc

```diff
@@ -183,6 +183,7 @@ StatusOr<ListBucketsResponse> RestStub::ListBuckets(
   if (!headers.ok()) return headers;
   request.AddOptionsToHttpRequest(builder);
   builder.AddQueryParameter("project", request.project_id());
+  builder.AddQueryParameter("pageToken", request.page_token());
```

Comment:
Does this work correctly for gRPC today? Can you also check that `ListObjects` functions correct.